From 953ef40b96ac611543838f27908fa69a38d33539 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin2.jansa@lgepartner.com>
Date: Fri, 27 Jan 2023 15:24:13 +0000
Subject: [PATCH] events.h: add missing functional include

:Release Notes:
Fix build.

:Detailed Notes:
Fixes:
http://gecko.lge.com/Errors/Details/527952
audiod/1.0.0-57-r32/git/include/events.h:183:14: error: 'function' in namespace 'std' does not name a template type
audiod/1.0.0-57-r32/git/src/modules/deviceManager/deviceManager.cpp:1128:71: error: 'struct events::EVENT_REQUEST_INTERNAL_DEVICES_INFO_T' has no member named 'func'
audiod/1.0.0-57-r32/git/src/modules/masterVolumeManager/OSEMasterVolumeManager.cpp:2425:35: error: 'struct events::EVENT_REQUEST_INTERNAL_DEVICES_INFO_T' has no member named 'func'

:Testing Performed:
Only build tested.

:QA Notes:
No change to image.

:Issues Addressed:
[WRP-6543] CCC: Various build fixes

Change-Id: Ia09aa6ae7728001c955005bae74494e244d233b4
---
Upstream-Status: Submitted [http://gpro.lge.com/c/webosose/audiod-pro/+/344272 events.h: add missing functional include ]

 include/events.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/events.h b/include/events.h
index 9f641ae3..b7004ce3 100644
--- a/include/events.h
+++ b/include/events.h
@@ -19,6 +19,7 @@
 #define _EVENTS_H_
 
 #include "utils.h"
+#include <functional>
 
 namespace events
 {
@@ -189,4 +190,4 @@ namespace events
     }EVENTS_T;
 }
 
-#endif //_EVENTS_H_
\ No newline at end of file
+#endif //_EVENTS_H_
